knr1050Sup/knr1050.proto

```diff
@@ -2,50 +2,45 @@ Terminator = CR;
 
 stop_pump {
     out "STOP:1,0";
-	in "OK";
+    in "OK";
 }
 
 stop_klv {
     out "STOP:2";
-	in "OK";
+    in "OK";
 }
 
 get_plim {
-	out "PLIM?";
-	in "PLIM:%(\$1)i,%(\$2)i";
+    out "PLIM?";
+    in "PLIM:%(\$1\$2)i,%(\$1\$3)i";
 }
 
 set_plim_low {
-	out "PLIM:%i,%(\$1)i";
-	in "OK";
+    out "PLIM:%i,%(\$1)i";
+    in "OK";
 }
 
 set_plim_high {
-	out "PLIM:%(\$1)i,%i";
-	in "OK";
+    out "PLIM:%(\$1)i,%i";
+    in "OK";
 }
 
 ramp {
-	out "RAMP:0,%(\$1\$2).0f,%(\$1\$3)i,%(\$1\$4)i,%(\$1\$5)i,%(\$1\$6)i";
-	in "OK"
+    out "RAMP:0,%(\$1\$2).0f,%(\$1\$3\$4)i,%(\$1\$3\$5)i,%(\$1\$3\$6)i,%(\$1\$3\$7)i";
+    in "OK"
 }
 
 get_status {
-	out "STATUS?";
-	in "STATUS:%(\$1.A)u,%(\$1.B)u,%(\$1.C)u,%*[^,],%(\$1.E)f,%(\$1.F)f,%(\$1.G)f,%(\$1.H)f,%(\$1.I)f,%*u,%*u,%*u,%*u,%*u,%*u,%*u,%*u,%(\$1.J)f,%(\$1.K)u,%*u";
+    out "STATUS?";
+    in "STATUS:%(\$1.A)u,%(\$1.B)u,%(\$1.C)u,%*[^,],%(\$1.E)f,%(\$1.F)f,%(\$1.G)f,%(\$1.H)f,%(\$1.I)f,%*u,%*u,%*u,%*u,%*u,%*u,%*u,%*u,%(\$1.J)f,%(\$1.K)u,%*u";
 }
 
 get_remote_mode {
-	out "REMOTE?";
-	in "REMOTE:%(\$1)i"
+    out "REMOTE?";
+    in "REMOTE:%(\$1)i"
 }
 
-set_remote_mode {
-	out "REMOTE";
-	in "OK";
-}
-
-set_local_mode {
-	out "LOCAL";
-	in "OK";
-}
+set_mode {
+    out "%{LOCAL|REMOTE}";
+    in "OK";
+}
```

Comment:
It is important to add a new line at the end of files read/written by EPICS, otherwise the line may not be read properly

Reply:
Sure! I've added the new line.